contracts/arbitrum/INodeInterface.sol

```diff
@@ -39,11 +39,8 @@ interface NodeInterface {
         uint256 deposit,
         address destAddr,
         uint256 l2CallValue,
-        uint256 maxSubmissionCost,
         address excessFeeRefundAddress,
         address callValueRefundAddress,
-        uint256 maxGas,
-        uint256 gasPriceBid,
         bytes memory data
     ) external view returns (uint256, uint256);
```

Comment:
I actually think this doesn't return a uint256 anymore - it looks like it doesn't return any numbers and the transaction gas itself is the estimate.

https://github.com/OffchainLabs/nitro/blob/98d635ca5a821681377f9ac25fbcae7727a6e67b/system_tests/retryable_test.go#L135

Reply:
https://github.com/OffchainLabs/nitro/blob/182e60083e321c01fd46aba64b3451ec49562fec/contracts/src/node-interface/NodeInterface.sol#L32

Reply:
Good catch. Thanks
Fixed the signature in 9136e05